src/Components/PaperEditor/TranscriptsContainer/index.js

```diff
@@ -52,7 +52,7 @@ const TranscriptsContainer = ({
         <TranscriptTabContent
           projectId={ projectId }
           transcriptId={ id }
-          labels={ labels }
+          labels={ labelsOptions }
```

Comment:
rename all the `labelsOptions` back to `labels`

Reply:
there are other areas that labelsOptions, I didn't mark all of them so please find them :)